standalone-metastore/metastore-server/src/main/java/org/apache/hadoop/hive/metastore/txn/CompactionTxnHandler.java

```diff
@@ -1430,6 +1427,44 @@ public void markRefused(CompactionInfo info) throws MetaException {
     updateStatus(info);
   }
 
+
+  @Override
+  public void retryCleanerAttemptWithBackoff(CompactionInfo info, long retentionTime) throws MetaException {
```

Comment:
The name sounds like this method is supposed to retry cleaning. Maybe setNextCleanerAttemptTime or something like that would be better?

Reply:
Renamed it to setCleanerRetryRetentionTimeOnError because in the new version it is responsible for updating the CQ_RETRY_RETENTION field in the database. Please note that this is the 4th name if this method, so if I have to rename it once more the new name will be doVeryImortantStuff()  :)